src/components/WeatherPanes/ListPane.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from 'assets/css/styles.css';
+import Store from 'Store';
+
+import Icon from 'components/reusable/icons/Icon';
+
+class ListPane extends React.PureComponent {
+  constructor(props){
+    super(props);
+    this.state = {}
+  }
+
+  componentDidMount() {
+    console.log('pep')
+    this.getTempUnit();
+  }
+
+  // componentWillUnmount() {
+  // }
+
+  getTempUnit() {
```

Comment:
I'd say this is not a **get** function but probably **subscribe** or **setListener** or **init** something ... what do you think ?